pom.xml

```diff
@@ -519,7 +519,7 @@
     <logging.logLevel>debug</logging.logLevel>
     <log4j2.version>2.6.2</log4j2.version>
     <enforcer.plugin.version>1.4.1</enforcer.plugin.version>
-    <junit.version>5.7.0</junit.version>
+    <junit.version>5.8.1</junit.version>
```

Comment:
I've updated it to see whether there is something like `@EnabledOnArch()` but then forgot to revert it.
Should I revert to 5.7.0 since it is not really related to this issue ?

Reply:
yeah just revert it ...